demo/full/scripts/components/Options/AudioAdaptiveSettings.jsx

```diff
@@ -51,35 +69,50 @@ function AudioAdaptiveSettings({
           <label htmlFor="initialAudioBitrate">Initial Audio Bitrate</label>
           <span className="wrapperInputWithResetBtn">
             <input
-              type="number"
+              type="text"
               name="initialAudioBitrate"
               id="initialAudioBitrate"
               aria-label="Initial audio bitrate option"
               placeholder="Number"
-              onChange={(evt) => onInitialAudioBrInput(evt.target.value)}
-              value={initialAudioBr}
+              onChange={(evt) => {
+                const { value } = evt.target;
+                updateInitialAudioBitrateText(value);
+                let newBitrate = value === "" ?
+                  defaultInitialAudioBitrate :
+                  parseFloat(value);
+                newBitrate = isNaN(newBitrate) ?
+                  defaultInitialAudioBitrate :
+                  newBitrate;
+                onInitialAudioBitrateChange(newBitrate);
+              }}
```

Comment:
Conversion between the text representation of the option (`initialAudioBitrateTxt`) and its communicated resulting number value (`initialAudioBitrate`) is now performed inside the corresponding component. It just made more sense to me as the text representation of that option should only be interesting to this component and nobody else.